resources/views/community/components/claims/finished-claims.blade.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use LegacyApp\Community\Enums\ClaimFilters;
+use LegacyApp\Community\Enums\ClaimSorting;
+
+$claimData = getFilteredClaims(
+    gameID: null,
+    claimFilter: ClaimFilters::AllCompletedPrimaryClaims,
+    sortType: ClaimSorting::FinishedDateDescending,
+    getExpiringOnly: false,
+    username: null,
+    offset: 0,
+    limit: $count
+);
+
+$allFinishedClaimsHref = '/claimlist.php?s=' . ClaimSorting::FinishedDateDescending . '&f=' . ClaimFilters::AllCompletedPrimaryClaims;
+?>
+
+<div class="component">
+    <h3>New Sets/Revisions</h3>
+
+    @if(!$claimData || count($claimData) === 0)
+        <div class="h-40 w-full flex flex-col items-center justify-center">
+            <img src="assets/images/cheevo/thinking.webp" alt="No new sets/revisions">
+            <p>Couldn't find any new sets/revisions.</p>
+        </div>
+    @else
+        <div>
+            <table class="table-highlight mb-1">
+                <thead>
+                    <tr class="do-not-highlight">
+                        <th colspan="2">Game</th>
+                        <th colspan="2">Dev</th>
+                        <th>Type</th>
+                        <th>Finished</th>
+                    </tr>
+                </thead>
+
+                <tbody>
+                    @foreach($claimData as $claim)
+                        @if (isValidConsoleId($claim['ConsoleID']))
+                            <x-claim-finished-table-row :claim="$claim" />
+                        @endif
```

Comment:
this `if` is what's causing my 6th items to disappear.

Reply:
Thank you for this additional context - the root issue should be resolved via https://github.com/RetroAchievements/RAWeb/commit/14f4b5d1023eb834da4600e580cd8031b498a3b9